src/arcade/patch/agent/cell/PatchCellFactory.java

```diff
@@ -159,80 +153,99 @@ public void createCells(Series series) {
+
+        return new PatchCellContainer(
+                id,
+                0,
+                pop,
+                age,
+                divisions,
+                State.UNDEFINED,
+                volume,
+                height,
+                volume,
+                height + compression);
     }
-    
+
     /**
      * Parses the population settings into maps to parameter value.
-     * <p>
-     * Loaded parameters include:
+     *
+     * <p>Loaded parameters include:
+     *
      * <ul>
-     *     <li>{@code CELL_VOLUME_MEAN} = cell volume distribution average</li>
-     *     <li>{@code CELL_VOLUME_STDEV} = cell volume distribution standard
-     *         deviation</li>
-     *     <li>{@code CELL_HEIGHT_MEAN} = cell height distribution average</li>
-     *     <li>{@code CELL_HEIGHT_STDEV} = cell height distribution standard
-     *         deviation</li>
-     *     <li>{@code CELL_AGE_MIN} = minimum cell age</li>
-     *     <li>{@code CELL_AGE_MAX} = maximum cell age</li>
-     *     <li>{@code DIVISION_POTENTIAL} = maximum number of divisions</li>
-     *     <li>{@code COMPRESSION_TOLERANCE} = maximum compression
-     *         tolerance</li>
+     *   <li>{@code CELL_VOLUME_MEAN} = cell volume distribution average
+     *   <li>{@code CELL_VOLUME_STDEV} = cell volume distribution standard deviation
+     *   <li>{@code CELL_HEIGHT_MEAN} = cell height distribution average
+     *   <li>{@code CELL_HEIGHT_STDEV} = cell height distribution standard deviation
+     *   <li>{@code CELL_AGE_MIN} = minimum cell age
+     *   <li>{@code CELL_AGE_MAX} = maximum cell age
+     *   <li>{@code DIVISION_POTENTIAL} = maximum number of divisions
+     *   <li>{@code COMPRESSION_TOLERANCE} = maximum compression tolerance
      * </ul>
      *
-     * @param series  the simulation series
+     * @param series the simulation series
      */
     void parseValues(Series series) {
         Set<String> keySet = series.populations.keySet();
-        
+
         for (String key : keySet) {
             MiniBox population = series.populations.get(key);
             int pop = population.getInt("CODE");
             popToIDs.put(pop, new HashSet<>());
-            
-            popToCriticalVolumes.put(pop, new Normal(population.getDouble("CELL_VOLUME_MEAN"),
-                    population.getDouble("CELL_VOLUME_STDEV"), random));
-            popToCriticalHeights.put(pop, new Normal(population.getDouble("CELL_HEIGHT_MEAN"),
-                    population.getDouble("CELL_HEIGHT_STDEV"), random));
-            popToAges.put(pop, new Uniform(population.getDouble("CELL_AGE_MIN"),
-                    population.getDouble("CELL_AGE_MAX"), random));
+
+            popToCriticalVolumes.put(
+                    pop,
+                    new Normal(
+                            population.getDouble("CELL_VOLUME_MEAN"),
+                            population.getDouble("CELL_VOLUME_STDEV"),
+                            random));
```

Comment:
this does not spark joy

Reply:
What would your preference for formatting this part be?